File: executable/r/stacked-bar.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  name = "SF Zoo", 
  x = c("giraffes", "orangutans", "monkeys"), 
  y = c(20, 14, 23), 
  type = "bar"
)
trace2 <- list(
  name = "LA Zoo", 
  x = c("giraffes", "orangutans", "monkeys"), 
  y = c(12, 18, 29), 
  type = "bar"
)

layout <- list(
  barmode = "stack", 
  xaxis = list(type = "category"), 
  categories = c("giraffes", "orangutans", "monkeys")
)

response <- p$plotly(trace0, trace1, kwargs=list(layout=layout, filename="stacked-bar", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/stacked-bar.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var trace0 = {
  x: ["giraffes", "orangutans", "monkeys"], 
  y: [20, 14, 23], 
  name: "SF Zoo", 
  type: "bar"
};
var trace1 = {
  x: ["giraffes", "orangutans", "monkeys"], 
  y: [12, 18, 29], 
  name: "LA Zoo", 
  type: "bar"
};
var data = [trace1, trace2];
var layout = {
  xaxis: {type: "category"}, 
  categories: ["giraffes", "orangutans", "monkeys"], 
  barmode: "stack"
};

plot_url = plotly.plot(data, layout, function (err, msg) {
    console.log(msg);
});